src/Common.php

```diff
@@ -23,7 +23,10 @@ class Common
 {
     protected Client $client;
     private const DEFAULT_BACKOFF_RETRIES = 10;
-    private const DEFAULT_HEADERS = ['Accept' => 'application/json'];
+    private const DEFAULT_HEADERS = [
+        'Accept' => 'application/json',
+        'Content-Type' => 'application/json',
```

Comment:
Nevim jak funguje migration endpoint, ale jinde to hlasi: 
```
{"error":"Content type \"\" is not supported, use \"application\/json\" or \"text\/plain\".","code":0,"exceptionId":"exception-1e0e0f1117a51ae28800bc38f19ef6f4","status":"error","context":[]}
```

Takze imho to doted nefungovalo.
Tim ze `Common` handluje request a response jako json encoded string, tak by to mohlo byt rovnou tady na jednom miste

Reply:
No ale zase bacha, ze `Content-Type` dava smysl jen s `POST` apod., `GET` nema content. Ted tu treba zadny `GET` request neni, ale jendou by to nekoho vypeklo (me to uz takle na nejakym klientovi vypeklo).

Reply:
Navic, ty chces posilat `text/plain`, ne? :trollface: Tak proc pridavat default?

Reply:
Hodil jsem to jen na POST a PATCH